src/navigation/AppNavigation/appNavigation.settings.ts

```diff
@@ -0,0 +1,11 @@
+import { DefaultTheme } from '@react-navigation/native';
+
+import { COLORS } from 'constants/color';
+
+export const APP_THEME = {
+  ...DefaultTheme,
+  colors: {
+    ...DefaultTheme.colors,
+    background: COLORS.light.primary[400],
```

Comment:
Please, delete background colour in screens styles, now we use theme for all screens which covered by navigation container